torch/csrc/jit/passes/onnx/preinline_onnx.cpp

```diff
@@ -0,0 +1,52 @@
+#include <torch/csrc/jit/passes/onnx/preinline_onnx.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+
+namespace torch {
+namespace jit {
+
+void replaceFunctions(Node* to_replace, Function* callee) {
+  if (callee->name() == "interpolate") {
```

Comment:
We should be matching the full qualified name here (`torch.nn.functional.interpolate`) instead of just `interpolate`. otherwise it will replace any user function named interpolate too, such as https://github.com/pytorch/vision/blob/24f16a338391d6f45aa6291c48eb6d5513771631/torchvision/ops/misc.py#L103.

Reply:
Got it. I'm not familiar with QualName structure but what do you think about a check like:

static auto namespaceName = std::string("___torch_mangle_3");
static auto qualFuncName = std::string("__torch__.torch.nn.functional.") + namespaceName + ".";

if (callee->qualname() == QualifiedName(qualFuncName + "interpolate")) {
...

Reply:
Yea, that seems about right. maybe something like: `fun_type->function()->qualname().qualifiedName()->find("torch.nn.functional.interpolate") != std::string::npos`